cmd/thor/flags.go

```diff
@@ -128,6 +128,11 @@ var (
 		Name:  "on-demand",
 		Usage: "create new block when there is pending transaction",
 	}
+	blockInterval = cli.IntFlag{
+		Name:  "interval",
```

Comment:
`block-interval` maybe? I think interval is too generic

Reply:
What about now?